Makefile

```diff
@@ -56,6 +56,8 @@ draconctl-image: cmd/draconctl/bin
 
 draconctl-image-publish: draconctl-image
 	$(DOCKER) push "${CONTAINER_REPO}/draconctl:${DRACON_VERSION}"
+	$(DOCKER) tag "${CONTAINER_REPO}/draconctl:${DRACON_VERSION}" "${CONTAINER_REPO}/draconctl:latest"
```

Comment:
@ptzianos  please take a look,
Due to rapid development i think having the ability to use this image from the latest tag always makes sesne

Reply:
if we publish it as the latest, we push the `v0.14.0` tag, it's published, then we trigger for some reason the v0.13.0 publish job because it failed and now we have a wrong reference to latest. I think we should push with both tags, so that at least we can pin stuff to the version where needed and play looser wherever we want

Reply:
that's exactly what it's doing, look one line below :D